CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ## [develop]
 
 ### Added
+- New `ListBox` UI element to display multiple selectable items
 - Icon for `Subtitles`
```

Comment:
`Subtitles` and `AudioTracks` are not componentos or other classes within the UI framework so they don't belong into `code` markup